Hunting Queries/AWSCloudTrail/AWS_PrivilegedRoleAttachedToInstance.yaml

```diff
@@ -2,7 +2,9 @@ id: 0db42a94-e7c8-4bf1-99a7-1a2fb4158212
 name: Privileged role attached to Instance
 description: |
   Identity and Access Management (IAM) securely manages access to AWS services and resources. 
-  This query looks for when Privileged role is attached to existing instance or new instance at deployment. This instance may be used by adversary to escalate normal user privileges to adminsitrative user.
+  Identifies when a Privileged role is attached to an existing instance or new instance at deployment. This instance may be used by an adversary to escalate a normal user privileges to an adminsitrative level.
```

Comment:
needs single quotes around just in case characters get updated

Reply:
updated